Move menu items into virtualization section

yaacov opened the pull request:
Move Forklift menu items to be under the virtualization section in the menu.

Forklift operators are moving from "Konveyor" project to be part of "Kubevirt" project, it makes sense that the menu items will also reflect this change.

If kubevirt plugin is not enabled:
  - add virtualization section
  - add import items below it

if kubevirt plugin is enabled:
 - add a separator ( after "migrationpolicies", witch is the last item in virtualization menu as of 4.12 )
 - add import items below the separator

Screenshots:
The movie:
![mtv+cnv](https://user-images.githubusercontent.com/2181522/191067281-d810f75e-c00a-4bf0-8c22-b54806a509d8.gif)

Before:
![vm-import-section](https://user-images.githubusercontent.com/2181522/190978455-90d40760-d7d5-4e8b-9910-13d5cce17cf7.png)

After:
![virt-section](https://user-images.githubusercontent.com/2181522/190978631-affeb761-8499-4c66-a443-3deb5db390ed.png)

After with kubevirt plugin running:
![screenshot-localhost_9000-2022 09 19-19_23_18](https://user-images.githubusercontent.com/2181522/191068057-9bc9c2db-a1be-41f7-914a-9b82e0238f9d.png)

Signed-off-by: yzamir <yzamir@redhat.com>

yaacov commented:
@vconzola please review

mturley commented:
I'll wait for @vconzola's input as well, but I'm not sure I can think of a better name for these items if they are present directly underneath Virtualization like this.

I think ideally if there was some way to add one more level of nesting, we could have Virtualization -> Import VMs -> Providers/Mappings/Plans. I know the Console doesn't support a tertiary nav like this currently, so maybe there is a way to have only one nav item (Virtualization -> Import VMs) and then have all of the Forklift views accessible from that one page. 

One way I can imagine we might be able to achieve that is by adding tabs at the top of the page for Providers / Mappings / Plans, which would require that we then eliminate the existing tabs somehow if we don't want two levels of tabs. The contents of the Providers and Mappings pages are small enough that perhaps we could just break those tabs out onto the same page in some multi-table layout (e.g. the Providers tab would have 3 tables on it stacked vertically: Kubevirt, oVirt, VMware. The Mappings tab would have 2 tables stacked vertically: Network and Storage)

I'm not the designer here so this is just me thinking out loud... @vconzola I'm curious if you have any ideas here

vconzola commented:
Just putting "Import" in front of "Providers", "Mappings" and "Plans" is misleading and will cause confusion. What else is under Virtualization in the navigation when kubevirt is installed - there must be more than just "Virtual Machines", right? 

Adding a tertiary navigation is definitely the way to go if it's acceptable to the OpenShift team and we have time to implement it. If not, I suggest changing the MTV nav items under Virtualization to "Providers for VM Import", "Mappings for VM Import" and "Plans for VM Import". Those are longer but more accurate and descriptive and will still fit on a single line.

yaacov commented:
> What else is under Virtualization in the navigation when kubevirt is installed - there must be more than just "Virtual Machines", right?

yes, just didn't have an openshift cluster with both (on k8s only virtualmachine showup, all the rest items are openshift only)
![screenshot-localhost_9000-2022 09 19-19_07_22](https://user-images.githubusercontent.com/2181522/191063089-033ae7ad-6105-47f8-b4bc-87a5f83e89d8.png)

yaacov commented:
@vconzola hi, thanks for the advices
a. I added a movie to the top description
b. I updated the menu items to be more accurate and descriptive.

please re-review

vconzola reviewed (commented):
Looks good to me. Thanks.
